rollup.config.js

```diff
@@ -6,7 +6,11 @@ export default {
     dest: 'build/webcharts.js',
     format: 'umd',
     globals: {d3: 'd3'},
-    external: ['d3'],
+    external: (function() {
```

Comment:
Is this needed? Is anything broken in the current build if we leave both `package.json` and `rollup.config.js` unchanged?